### ISSUE TITLE
Number of parameters

### ISSUE THREAD

**Issue by ducha-aiki** [forestagostinelli/Learned-Activation-Functions-Source#4]
Hi,

If I understand source code correctly
https://github.com/forestagostinelli/Learned-Activation-Functions-Source/blob/master/src/caffe/layers/apl_layer.cpp#L71
and 
https://github.com/forestagostinelli/Learned-Activation-Functions-Source/blob/master/src/caffe/layers/apl_layer.cpp#L30

then when used in convolution layers, APL learns not C_sums paramers, but H_W_C (== count/num) \* sums parameters. So same channel in different locations does not share same APL coefficients.  => APL with sums = 2 has h_w more parametrs than PReLU.
Is it intentional or typo?

**Comment by forestagostinelli** [comment]
Yes, that was the initial design. We are working on a new version that only
takes up C*sums or even just sums number of parameters.
